### PR TITLE
upup/pkg/fi/cloudup/awstasks/ staticcheck

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/kubernetes/kops/issues/7800

### CODE REVIEW

**Comment by tanjunchen** [comment]
/cc @rifelpet
@hakman
for review

**Comment by hakman** [comment]
I think this clashes with this https://github.com/kubernetes/kops/pull/8193.
Maybe should wait with with the formatting fixes until the last few files are removed from `hack/.staticcheck_failures`.

File: upup/pkg/fi/cloudup/apply_cluster.go [hunk 1]
```diff
@@ -1024,7 +1024,7 @@ func (c *ApplyClusterCmd) validateKopsVersion() error {
 		fmt.Println("")
 		fmt.Printf("More information: %s\n", buildPermalink("upgrade_kops", recommended.String()))
 		fmt.Println("")
-		fmt.Printf(starline)
+		fmt.Println(starline)
```
Comment (justinsb) [inline upup/pkg/fi/cloudup/apply_cluster.go]: Sorry - I think I just approved an earlier PR that is now conflicting here.  You might want to rebase / drop the changes in this file, as I think this file now passes staticcheck.

**Comment by justinsb** [comment]
Sorry - I think I just created a conflict on this file (I've been working through the PR backlog in submission order).  But if you rebase I think you can drop the changes to `apply_cluster.go`, and the other changes LGTM - thanks @tanjunchen 

/approve
/lgtm

File: upup/pkg/fi/cloudup/apply_cluster.go (hunk 1 above)
Comment (tanjunchen) [reply upup/pkg/fi/cloudup/apply_cluster.go]: no matter.
i will update it.
thanks

File: upup/pkg/kutil/import_cluster.go [hunk 2]
```diff
@@ -507,18 +506,18 @@ func (x *ImportCluster) ImportAWSCluster() error {
 	return nil
 }
 
-func parseInt(s string) (int, error) {
```
Comment (tanjunchen) [inline upup/pkg/kutil/import_cluster.go]: Some commented files refer to this function. So comment this function not remove it

**Comment by tanjunchen** [comment]
@johngmyers 
thanks review

**Pull request merged**